docs/pages/prerequisites.md

```diff
@@ -0,0 +1,27 @@
+# Prerequisites
+
+To get started with Fable, you will need a couple of things installed on your machine. 
+
+ - F# Compiler
+ - Node (with Npm)
+ - Yarn
+ - Dotnet SDK
+ - IDE
+
+## F# Compiler
+If you have Visual Studio or Xamarin installed with Visual F# Tools installed then you already have F# Compiler on your machine. If you are starting from scratch, refer to [fsharp.org](http://fsharp.org/) to get started with F#. 
```

Comment:
I'd remove this as it's not necessary to install F# compiler in the machine for Fable (it comes already embedded).